src/index.html

```diff
@@ -8,6 +8,73 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars--star"></div>
+
+        <div class="stars--star"></div>
+
+        <div class="stars--star"></div>
+
+        <div class="stars--star"></div>
+
+        <div class="stars--star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars--star"></div>
+
+        <div class="stars--star"></div>
+
+        <div class="stars--star"></div>
+
+        <div class="stars--star"></div>
+
+        <div class="stars--star"></div>
+      </div>
```

Comment:
```suggestion
      <div class="stars stars--0">
        <div class="stars--star"></div>
        <div class="stars--star"></div>
        <div class="stars--star"></div>
        <div class="stars--star"></div>
        <div class="stars--star"></div>
      </div>
      
      <div class="stars stars--1">
        <div class="stars--star"></div>
        <div class="stars--star"></div>
        <div class="stars--star"></div>
        <div class="stars--star"></div>
        <div class="stars--star"></div>
      </div>
```

Reply:
It's a good practice to split big blocks of code by empty lines, but no need to split every element it's not increasing readability